lib/messaging/message/metadata.rb

```diff
@@ -60,12 +61,8 @@ def follow(preceding_metadata)
 
         self.reply_stream_name = preceding_metadata.reply_stream_name
 
-        preceding_metadata.properties.each do |property|
-          if property.local?
-            next
-          end
-
-          properties << property.dup
+        preceding_metadata.properties.each do |name, value|
```

Comment:
Should metadata.follow overwrite properties or add to it? In other words, if the destination metadata already has a property `:foo` but the source only has a property `:bar`, would you expect the metadata after a follow to have both `:foo` and `:bar`, or just `:bar`?

Reply:
Leaving this as is.